manifold/mnist_manifold.py

```diff
@@ -18,17 +18,20 @@
 
 import os
 
-#########
-## Loads  pretrained MNIST model
-## Learns manifold on data
-## Creates adversarial images
-## Tests adversarial images on model
-#########
+'''
+Loads  pretrained MNIST model
+Loads manifold from file
+Creates adversarial image
+Tests adversarial image on model
+Displays results
+'''
 
 WIDTH = 28
 HEIGHT = 28
 EPSILON = 0.15  # example values: low = .01, high = .15
 PATH_TO_MODEL = "models/mnist_model.h5"
+MANIFOLD_FILE = "mnist-lle-60000-2d-2"
+MANIFOLD_SIZE = 60000  # The number of points in the manifold; used for visalization colors
 
 
 def convert_to_model(seq_model):
```

Comment:
do we need convert_to_model() in two places, or could we import?

Reply:
Yes, the manifold stuff all still runs.
I separated out the convert_to_model() and the color list into mnist_tools. See what you think.

Reply:
looks good to me 👍